hubblestack/extmods/hubble_mods/osquery.py

```diff
@@ -0,0 +1,217 @@
+'''
+Flexible Data Gathering: osquery
+
+This module allows for running osquery queries
+
+Mandatory param:
+    query -  The query which needs to be run
+
+Optional param:
+    cast_to_string - Default (False) - Specifies if the command output (ex. list of dictionaries)  needs to be type cast to string
+
+    osquery_args - string of args to pass to osquery. Note that the ``--read_max`` and ``--json`` args are already included.
```

Comment:
We should rename it to flags, just like we did in grep.

Reply:
done